<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNetwork</class>
 <widget class="QDialog" name="DialogNetwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QRadioButton" name="serverRadio">
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Server IP Address</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="portEdit"/>
   </item>
   <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QRadioButton" name="clientRadio">
     <property name="text">
      <string>Client</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="addressEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Confim</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
